components/sensor/ezo.rst

```diff
@@ -43,20 +45,146 @@ Configuration variables:
+  .. code-block:: cpp
+
+      id(ph_ezo).get_state();
+
+
+- ``get_slope()``: Sensor retrieves slope and triggers ``on_slope:`` once done
+
+  .. code-block:: cpp
+
+      id(ph_ezo).get_slope();
+
+
+- ``get_t()``: Sensor retrieves temperature compensation value (in Celcius) and triggers ``on_t:`` once done
+
+  .. code-block:: cpp
+
+      id(ph_ezo).get_t();
+
+
+- ``set_t(float value)``: Send the given temperature (in Celcius) to the sensor.
+
+  .. code-block:: cpp
+
+      id(ph_ezo).set_t("27.00");
+
+
+- ``set_tempcomp_value(float temp)``: Send the given temperature (in Celcius) to the sensor (this is an alias of `set_t()` for backwards compatibility)
 
   .. code-block:: cpp
 
-      // Within a lambda, set the temperature compensation value from the temperature sensor
       id(ph_ezo).set_tempcomp_value(id(rtd_ezo).state);
 
 
+- ``get_calibration()``: Sensor retrieves calibration and triggers ``on_calibration:`` once done
+
+  .. code-block:: cpp
+
+      id(ph_ezo).get_calibration();
+
+
+- ``set_calibration_generic(float value)``: Sets the calibration with no point.
+
+  .. code-block:: cpp
+
+      id(ph_ezo).set_calibration_generic(750.0);
+
+
+- ``set_calibration_point_low(float value)``: Sets the low calibration point.
+
+  .. code-block:: cpp
+
+      id(ph_ezo).set_calibration_point_low(4.00);
+
+
+- ``set_calibration_point_mid(float value)``: Sets the medium calibration point.
+
+  .. code-block:: cpp
+
+      id(ph_ezo).set_calibration_point_mid(7.00);
+
+
+- ``set_calibration_point_high(float value)``: Sets the high calibration point.
+
+  .. code-block:: cpp
+
+      id(ph_ezo).set_calibration_point_low(10.00);
+
+
+- ``clear_calibration()``: Clears all calibration points.
+
+  .. code-block:: cpp
+
+      id(ph_ezo).clear_calibration();
+
+
+- ``get_led_state()``: Sensor LED state and triggers ``on_led:`` once done
+
+  .. code-block:: cpp
+
+      id(ph_ezo).get_led_state();
+
+
+- ``set_led_state(bool on)``: Sensor LED on or off
+
+  .. code-block:: cpp
+
+      id(ph_ezo).set_led_state(true);
+
+
+- ``send_custom(const std::string &payload, uint16_t delay_ms = 300, bool response_expected = false)``: Runs a custom command. This sends exactly what is in ``payload``. Optionally you can set a ``delay`` and if a response is expected that should be parsed. Defaults to ``false`` for custom commands.
+
+  .. code-block:: cpp
+
+      // Run a custom command to turn on the LED
+      id(ph_ezo).send_custom("L,1");
+
+
+.. _evo_callbacks:
+
+Callbacks
+---------
+
+- **on_led:** : Triggered when the result of ``get_led_state()`` is ready
+- **on_device_information:** : Triggered when the result of ``get_device_information()`` is ready
+- **on_slope:** : Triggered when the result of ``get_slope()`` is ready
+- **on_calibration:** : Triggered when the result of ``get_calibration()`` is ready
+- **on_t:** : Triggered when the result of ``get_t()`` is ready
+- **on_custom:** : Triggered when the result of ``get_custom()`` is ready
```

Comment:
These should be part of the above configuration variables list in the correct format.